ios/brave-ios/Sources/PlaylistUI/PlayerModel.swift

```diff
@@ -558,7 +602,7 @@ public final class PlayerModel: ObservableObject {
       case .loadingStreamingURLFailed(.cannotLoadMedia):
         return Strings.PlayList.sorryAlertTitle
       default:
-        return "Something went wrong"
+        return "Something went wrong"  // TODO: Localize
```

Comment:
Maybe you might wanna mark other places with a similar TODO, there is come and they are easy to miss in the end :D